### PR TITLE
node-labeller: add support for emulation

### PR DESCRIPTION
**Pull request by jean-edouard**
**What this PR does / why we need it**:
Without this PR, on a cluster that doesn't support hardware-assisted virtualization, VMs with dedicated CPUs will fail to start.
The virt-launcher errors look like: "failed to format domain cputune.", "not enough exclusive threads provided".
This is because node-labeller didn't populate `/var/lib/kubevirt-node-labeller` with things like the host CPU topology.
I don't see why node labelling or CPU pinning should be incompatible with emulation, but please let me know if I'm missing something.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
CPU pinning doesn't require hardware-assisted virtualization anymore
```

### CODE REVIEW

File: cmd/virt-launcher/node-labeller/node-labeller.sh [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
+KVM_MINOR=$(grep -w 'kvm' /proc/misc | cut -f 1 -d' ')
+VIRTTYPE=qemu
+
+if [ ! -e /dev/kvm ] && [ -n "$KVM_MINOR" ]; then
+  mknod /dev/kvm c 10 $KVM_MINOR
 fi
 
-if [ ! -e /dev/kvm ]; then
-  mknod /dev/kvm c 10 $(grep '\\<kvm\\>' /proc/misc | cut -f 1 -d' ')
+if [ -e /dev/kvm ]; then
```
Comment (maya-r) [inline cmd/virt-launcher/node-labeller/node-labeller.sh:12]: Minor nit: I feel like we can combine this into a single conditional for simplicity sake.

Comment (jean-edouard) [reply cmd/virt-launcher/node-labeller/node-labeller.sh:12]: I'm not sure I see it, could you maybe suggest the specific code change?

Comment (maya-r) [reply cmd/virt-launcher/node-labeller/node-labeller.sh:12]: nvm I misread

**Comment by vladikr** [comment]
/approve
Thanks, @jean-edouard - at least we shouldn't crash as before.

**Pull request merged**